spec/periodicChecksSpec.js

```diff
@@ -717,144 +717,144 @@ describe('Periodic Checks Module', () => {
       };
     });
 
-    describe('When all issues have been added to a project', () => {
```

Comment:
Instead of this, add `/* istanbul ignore next */` just before line 619. And change the `describe` there to `xdescribe`